library/src/components/Schema.tsx

```diff
@@ -1,11 +1,13 @@
+/* eslint-disable sonarjs/cognitive-complexity */
```

Comment:
I think it works only if you put it right before the affected function

Reply:
Eslint doesn't work in any form, because then we should have errors from `lint` stage, and we don't.